packages/core/src/context/experiment/validation.ts

```diff
@@ -41,7 +41,8 @@ export const validateUpperBoundary = (experiment: ExperimentType): number[] => {
       )?.max
       if (
         valueVarMax !== undefined &&
-        typeof d.value === 'number' &&
+        d.value !== undefined &&
```

Comment:
Number check breaks validation because values are strings

Reply:
and undefined check cause type errors :)

Reply:
What about

```suggestion
        d.value !== undefined && ... && Number(d.value) > valueVarMax
```

Reply:
That's better. I have changed upper and lower boundary validations to use Number